crates/consensus-logic/src/checkpoint_verification.rs

```diff
@@ -112,5 +113,11 @@ pub fn verify_proof(
         return Ok(());
     }
 
+    if !accept_empty_proof && is_empty_proof {
+        return Err(ZkVmError::ProofVerificationError(
+            "Empty proof received, which is not acceptable in strict proof mode".to_string(),
```

Comment:
Can we have also an active voice recommendation here.
Something like "check the environmental variable FOO" or "use BAR" etc?